src/components/BottomSheet.tsx

```diff
@@ -47,6 +48,18 @@ const BottomSheet = ({
     []
   )
 
+  // fires before bottom sheet animation starts
+  const handleAnimate = (fromIndex: number, toIndex: number) => {
+    if (toIndex === -1 || fromIndex === -1) {
+      Keyboard.dismiss()
+    }
+  }
+
+  const handleClose = () => {
+    onClose?.()
+    scrollViewRef.current?.scrollTo({ y: 0, animated: false })
+  }
```

Comment:
Could we add comments as to why we're doing these? what's in the PR description.
It helps a lot while reading the code.